src/animated/Controller.ts

```diff
@@ -159,17 +165,21 @@ class Controller<DS extends object = any> {
     this.props = {} as any
     this.merged = {} as any
     this.values = {} as any
-    this.interpolations = {}
+    this.interpolations = {} as any
     this.animations = {}
     this.configs = []
   }
 
   // Add this controller to the frameloop
   private _start(onEnd?: OnEnd) {
-    if (onEnd) this.onEndQueue.push(onEnd)
-    if (this.idle) {
-      this.idle = false
-      start(this)
+    if (this.configs.length) {
+      if (onEnd) this.onEndQueue.push(onEnd)
+      if (this.idle) {
+        this.idle = false
+        start(this)
+      }
+    } else if (onEnd) {
+      onEnd(true)
```

Comment:
The `start` method now calls its `onEnd` argument immediately if no animations exist.

Idk if this edge case matters, but I decided to put it in for posterity.